matrix-room.c

```diff
@@ -1020,10 +1020,8 @@ void matrix_room_send_message(PurpleConversation *conv, const gchar *message)
         return;
```

Comment:
still leaks memory here.

In fact it looks like we shouldn't be doing the unescaping on this path?